spec/datadog/tracing/contrib/active_support/notifications/subscription_spec.rb

```diff
@@ -11,7 +11,7 @@
     subject(:subscription) { described_class.new(span_name, options, &block) }
 
     let(:span_name) { double('span_name') }
-    let(:options) { {} }
+    let(:options) { { resource: 'dummy_resource' } }
```

Comment:
Is this change also necessary?

Reply:
Yes -- see https://github.com/DataDog/dd-trace-rb/pull/2346/commits/158f165cc21966fa46b191d4c3f187f51801da4a for why :)